dbt-bigquery/src/dbt/adapters/bigquery/connections.py

```diff
@@ -572,7 +572,8 @@ def _query_and_results(
         query_job = client.query(
             query=sql,
             job_config=query_job_config,
-            job_id=job_id,  # note, this disables retry since the job_id will have been used
+            job_id=job_id,
+            job_retry=None,
```

Comment:
Altering this seems a o k. We're going to need to change the unit test responsible for failing by adding this to the profile generation.

Reply:
@colin-rogers-dbt I'm pretty sure that if you read the outpout note, us doing the `job_id` is fine. Are you suggesting we should change that aspect to prefix?